src/plugins/uploader.js

```diff
@@ -0,0 +1,146 @@
+"use strict";
+
+const SocketIOFileUploadServer = require("socketio-file-upload/server");
+const Helper = require("../helper");
+const path = require("path");
+const fsextra = require("fs-extra");
+const fs = require("fs");
+const fileType = require("file-type");
+const readChunk = require("read-chunk");
+const crypto = require("crypto");
+const isUtf8 = require("is-utf8");
+const log = require("../log");
+
+const whitelist = [
```

Comment:
The plan is to open up this whitelist later on, or allow it to be turned off, yeah?

Reply:
Up for discussion. It's tied to the plugin, so future plugins can use their own.

Reply:
Yeah, I wasn't meaning for right now, but it can be a discussion later. Cool beans.